app/assets/javascripts/analytics-enhancedEcommerce.js

```diff
@@ -7,7 +7,13 @@ var enhancedEcommerceTracking = function (d) {
   // that element.
   var ancestor = function (el, selector) {
     // travel two up the tree to search one up the tree
-    var present = el.parentElement.querySelectorAll(selector)
+    var present = el.parentElement
+    
+    if (present) {
+      present = present.querySelectorAll(selector)
+    } else {
+      return null 
+    }
 
     if (present.length < 1) {
       return ancestor(el.parentElement, selector)
```

Comment:
Just noticed on line 42:

```javascript
var subsection = ancestor(listItem, '[data-ec-list-subsection]').getAttribute('data-ec-list-subsection')
```

will need a check to make sure that `.getAttribute()` isn't used when `null` is returned from `ancestor()`.